docs/en/ingest-management/fleet/fleet-settings.asciidoc

```diff
@@ -99,3 +137,8 @@ not currently provide validation.
 --
 
 . Save and apply the settings.
+
+//REVIEWERS: Should I say anything about the selections for making the
```

Comment:
Reviewers: Please respond to this question.

Reply:
In 8.0 this will not be editable so there is probably nothing to document